docs/CHANGELOG.rst

```diff
@@ -64,6 +64,7 @@ Fixed
 Added
 -----
 - Add bcftools version 1.14 to ``resolwebio/common`` Docker image
+- Add filters for species, tissue, treatment and outcome on BioCollection
```

Comment:
I would add something more descriptive and avoid using non-existing term BioCollection. Maybe something like 'Add ExtendedCollectionFilter filter to allow filtering collections by samples containing given species, tissue ...'.